lib/cpp/Kconfig

```diff
@@ -67,13 +116,7 @@ config CPP_MAIN
 	  and any applications defining its main() in a C++ source file must
 	  enable this option.
 
-config LIB_CPLUSPLUS
-	bool "Link with STD C++ library"
-	select REQUIRES_FULL_LIBC
-	help
-	  Link with STD C++ Library.
-
-if LIB_CPLUSPLUS
+if !MINIMAL_LIBCPP
 
 config CPP_EXCEPTIONS
```

Comment:
should these options be prefixed with `MINIMAL_LIB` to make it clear what they refer to?